infrastructure_source_code.md

```diff
@@ -0,0 +1,44 @@
+# DataWorks Open Source Infrastructure Source Code Policy
+
+## Purpose
+
+This document sets out local policy, for the Department for Work and Pensions (DWP) DataWorks team, on what types of information can and cannot be included in infrastructure code that is made publicly available.
+
+It is expected that, at some point, this document will be superceded by a DWP-wide open-source infrastructure code policy which encompasses all DWP public GitHub repositories. Some useful [DWP guidance on creating open source products](https://confluence.service.dwpcloud.uk/pages/viewpage.action?title=Guidance+on+creating+open+source+products&spaceKey=EN) currently exists, but it has not yet been made open-source.
+
+## Scope
+
+This policy covers infrastructure code. Please refer to the [index](README.md#index) for links to other related polices.
+
+## General Approach
+
+Infrastructure code should be made open wherever it is appropriate to do so:
+
+This approach makes code more readily accessible to a broad target audience, so that they can be more easily used for their intended purpose;
+1. This approach enables and encourages re-use of common standards across the DWP and other Government departments, which in turn makes collaboration easier. It also enables other parties to feed back improvements, which benefits everyone.
+1. This policy is in line with the spirit of the  [Government Digital Service (GDS) guidelines on making information open-source](https://gds-operations.github.io/guidelines/).
+
+## Security Restrictions
```

Comment:
The DPA limitation is also potentially relevant for Infrastructure code (our users and devs are covered by this, not just the general public. I note that there will be some common restrictions which encompass all of: documentation; infrastructure code; and application code (legal, corporate and commercial restrictions will likely be the same for all, even if the specific security restrictions might have some different nuances). Perhaps we need to think about refactoring the document set at some point, to minimise repetition? It's a different balance between minimising duplication and trying to make each document as self-contained as possible.

Reply:
I agree that some elements should be moved to the parent document (README.md) rather than duplicating in each child document